understanding/22/target-size-minimum.html

```diff
@@ -40,15 +40,15 @@ <h2>In brief</h2>
       <dl>
           <dt>Objective</dt><dd>Make controls easier to activate</dd>
           <dt>Author task</dt><dd>Ensure targets meet a minimum size or have sufficient spacing around them</dd>
-          <dt>Key beneficiaries</dt><dd>Users with some physical disabilities</dd>
+          <dt>Key beneficiaries</dt><dd>Users with limitations to fine motor input</dd>
```

Comment:
@bruce-usab I tried to keep the beneficiaries quite generic in the 'In brief' material, both for speed (there is a fairly small set of  'canned' beneficiaries to draw from) and so that anyone reading would likely understand what it meant.
I am mildly concerned that if we start to be more specific for this SC, we set ourselves up for a lot of nitpicking on the precision of other Key beneficiary statements. To me, the main Understanding document is where more particulars for different users can be detailed (which you have done). I'd advocate to leave this line as is for now.

Reply:
@bruce-usab you had marked this as resolved without responding. It might help to have the context of #3191 
```suggestion
          <dt>Key beneficiaries</dt><dd>Users with some physical disabilities</dd>
```